### PR TITLE
Update Japanese translation

### PR DESCRIPTION
**Pull request by momo-i**

### CODE REVIEW

File: src/main/resources/assets/securitycraft/lang/ja_jp.json [hunk 1]
```diff
@@ -578,8 +604,17 @@
 	"block.securitycraft.username_logger": "ユーザー名記録装置",
 	"commands.securitycraft.bug": "バグの報告はDiscordサーバーの#bugreportチャンネルでお願いします。最初にルールをお読みください。",
 	"commands.securitycraft.connect": "IRC チャットは、SecurityCraft の v1.8.3 でサポートされなくなりました。ヘルプが必要な場合は、SecurityCraft の Discord の #help チャンネルを参照してください。",
+	"commands.securitycraft.convert.fill.failed": "指定されたエリアに変換可能なブロックはありません",
+	"commands.securitycraft.convert.fill.success": "%s個のブロックの変換に成功しました",
+	"commands.securitycraft.convert.set.failed": "指定された位置に変換可能なブロックはありません",
+	"commands.securitycraft.convert.set.success": "%s, %s, %sのブロックを変換しました",
 	"commands.securitycraft.dump.notFound": "SecurityCraftでは\"%s\"に登録されているものはありません。",
 	"commands.securitycraft.dump.result": "ここをクリックすると、%sレジストリエントリをクリップボードにコピーできます。",
+	"commands.securitycraft.help": "%s: %s + %s",
```
Comment (bl4ckscor3) [inline src/main/resources/assets/securitycraft/lang/ja_jp.json:613]: Unnecessary because it is unchanged from en_us

**Review by bl4ckscor3 — CHANGES_REQUESTED** [review]
Thank you for the update! There are a few things that were changed which aren't addressed in your PR:

The wire cutters' help entry got changed to:
- `"help.securitycraft.wire_cutters.info": "The wire cutters can be used to defuse several types of mines and deactivate a cage trap by rightclicking the block with them. Furthermore, sneak-rightclicking a passcode-protected block removes the passcode protection.",`

This message got "at %3$s" added to it at the end. It will show the coordinates the portable radar is at:
- `"messages.securitycraft:portableRadar.withName": "%1$s is near your portable radar named %2$s at %3$s."`

The following messages were unchanged apart from adding "Set to 0 to toggle the signal" to the end:
- `"option.generic.signalLength.description": "How long should the redstone signal upon successful activation be? (In ticks, 20 ticks = 1 second. Set to 0 to toggle the signal)",`
- `"option.securitycraft.block_change_detector.signalLength.description": "How long should the redstone signal after detecting a block change be? (In ticks, 20 ticks = 1 second. Set to 0 to toggle the signal)",`
- `"option.securitycraft.rift_stabilizer.signalLength.description": "How long should the redstone signal after detecting a teleportation attempt be? (In ticks, 20 ticks = 1 second. Set to 0 to toggle the signal)",`
- `"option.securitycraft.sonic_security_system.signalLength.description": "How long should the sonic security system unlock blocks/give off a redstone signal after hearing the correct tune? (In ticks, 20 ticks = 1 second. Set to 0 to toggle the signal)",`

File: src/main/resources/assets/securitycraft/lang/ja_jp.json (hunk 1 above)
Comment (momo-i) [reply src/main/resources/assets/securitycraft/lang/ja_jp.json]: Thank you for reply. Fixed these changes.

**Review by bl4ckscor3 — APPROVED** [review]
Thank you very much!

**Pull request merged**